contract-addresses/mainnetAddresses.ts

```diff
@@ -0,0 +1,261 @@
+    address: '0xe1662531aA5de1DAD8ab5B5756b8F6c8F3C759Ca'
+  },
+  rariPool26Fei: { artifactName: 'CErc20Delegator', address: '0x38ee94FcF276Cee403f4645341f80e671d25b352' },
+  rariPool26FeiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0xFdCc96967C86250f333cE52Ba706Ec2961c3302f'
+  },
+  rariPool27Fei: { artifactName: 'CErc20Delegator', address: '0xda396c927e3e6BEf77A98f372CE431b49EdEc43D' },
+  rariPool27FeiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0x91f50E3183a8CC30D2A981C3aFA85A2Bf6691c67'
+  },
+  rariPool6Fei: { artifactName: 'CErc20Delegator', address: '0x185Ab80A77D362447415a5B347D7CD86ecaCC87C' },
+  rariPool6FeiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0xB51f09B6F103D697dc5d64DC904Ad6a2Dad39987'
+  },
+  rariPool7Fei: { artifactName: 'CErc20Delegator', address: '0xE640E9beC342B86266B2bD79F3847e7958cb30C4' },
+  rariPool7FeiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0x74B235Fef146cDB5BE0D3786a9f3774674b3615E'
+  },
+  rariPool8Comptroller: { artifactName: 'Unitroller', address: '0xc54172e34046c1653d1920d40333dd358c7a1af4' },
+  rariPool8Dai: { artifactName: 'CErc20Delegator', address: '0x7e9cE3CAa9910cc048590801e64174957Ed41d43' },
+  rariPool8DaiIrm: { artifactName: 'unknown', address: '0xede47399e2aa8f076d40dc52896331cba8bd40f7' },
+  rariPool8Eth: { artifactName: 'CErc20Delegator', address: '0xbB025D470162CC5eA24daF7d4566064EE7f5F111' },
+  rariPool8EthIrm: { artifactName: 'unknown', address: '0xbab47e4b692195bf064923178a90ef999a15f819' },
+  rariPool8Fei: { artifactName: 'CErc20Delegator', address: '0xd8553552f8868C1Ef160eEdf031cF0BCf9686945' },
+  rariPool8FeiIrm: { artifactName: 'unknown', address: '0x8f47be5692180079931e2f983db6996647aba0a5' },
+  rariPool8FeiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0x37349d9cc523D28e6aBFC03fc5F44879bC8BfFD9'
+  },
+  rariPool8Tribe: { artifactName: 'CErc20Delegator', address: '0xFd3300A9a74b3250F1b2AbC12B47611171910b07' },
+  rariPool8TribeIrm: { artifactName: 'unknown', address: '0x075538650a9c69ac8019507a7dd1bd879b12c1d7' },
+  rariPool9Fei: { artifactName: 'CErc20Delegator', address: '0x11A9F6ae6B36B4b820777D05B90Cd6CCCB1CDa31' },
+  rariPool9FeiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0xF2D8beE45f29A779cFB9F04ac233E703974a2C53'
+  },
+  rariPool9Rai: { artifactName: 'CErc20Delegator', address: '0x752F119bD4Ee2342CE35E2351648d21962c7CAfE' },
+  rariPool9RaiPCVDeposit: {
+    artifactName: 'ERC20CompoundPCVDeposit',
+    address: '0x9aAdFfe00eAe6d8e59bB4F7787C6b99388A6960D'
+  },
+  ratioPCVController: { artifactName: 'RatioPCVController', address: '0xfC1aD6eb84351597cD3b9B65179633697d65B920' },
+  reflexerStableAssetFusePoolRai: {
+    artifactName: 'CErc20Delegator',
+    address: '0x752F119bD4Ee2342CE35E2351648d21962c7CAfE'
+  },
+  reflexerStableAssetFusePoolRaiPCVDeposit: {
```

Comment:
What kind of linter is being used here? We should get everyone on the team set up with the same linter so all of our code conforms to a standard style.

Reply:
`npm run lint:fix` and `npm run prettier-format`